src/stacks/vue-js/index.ts

```diff
@@ -7,7 +7,10 @@ export class VueJS {
   async isAvailable(): Promise<boolean> {
     try {
       const packageJson = require(`${this.rootPath}/package.json`);
-      return Object.keys(packageJson.dependencies).includes('vue');
+      if (packageJson.dependencies !== undefined) {
```

Comment:
We should maybe create an utility method in order to check if the project as something as a dep or devDep